client/me/help/help-contact-form/index.jsx

```diff
@@ -77,7 +77,7 @@ export const HelpContactForm = React.createClass( {
 			howYouFeel: 'unspecified',
 			message: '',
 			subject: '',
-			siteSlug: this.getSiteSlug()
+			siteSlug: this.getSiteId()
```

Comment:
Hi @gwwar,

I just noticed an error in this PR. I'm sorry.
It should have been:

```js
siteId: this.getSiteId()
```

I didn't noticed it before cause it does not seems to cause any visual regression on master.
However it's important that we fix it, cause it breaks the `sites-dropdown` component in the contact form on the PR #9624.

I'm reopening this branch, and submit the fix.
Let me know if this is not the correct process.

Again, I'm sorry for the mistake.